src/recordlinker/routes/patient_router.py

```diff
@@ -65,6 +66,73 @@ def update_person(
         patient_reference_id=patient.reference_id, person_reference_id=person.reference_id
     )
 
+
+@router.post(
+    "/",
+    summary="Create a patient record and link to an existing person",
+    status_code=fastapi.status.HTTP_201_CREATED,
+)
+def create_patient(
+    payload: typing.Annotated[schemas.PatientCreatePayload, fastapi.Body],
+    session: orm.Session = fastapi.Depends(get_session),
+) -> schemas.PatientRef:
+    """
+    Create a new patient record in the MPI
```

Comment:
```suggestion
    Create a new patient record in the MPI and link to an existing person.
```

I was initially confused by this description because it doesn't imply that you need a `person_reference_id`. I thought this was a function that would just insert a new patient and assign them a `person_reference_id` (which we obviously should do but that's what I thought would happen based on the description alone).